src/parcsr_mv/par_csr_matop.c

```diff
@@ -6218,26 +6219,20 @@ hypre_ParCSRMatrixDropSmallEntries( hypre_ParCSRMatrix *A,
       return hypre_error_flag;
    }
 
-   hypre_GpuProfilingPushRange("ParCSRMatrixDropSmallEntries");
```

Comment:
@victorapm why is this no longer needed?

Reply:
It's already inside the `Device` function and no longer needed here